add-on/panels/js/roomViews.jsx

```diff
@@ -412,8 +412,14 @@ loop.roomViews = (function(mozL10n) {
      */
     leaveRoom: function() {
       if (this.state.used) {
+        // The room has be used, so we might want to display the feedback view.
+        // Therefore we leave the room to ensure that we have stopped sharing etc,
+        // then trigger the call terminated handler that'll do the right thing
+        // for the feedback view.
         this.props.dispatcher.dispatch(new sharedActions.LeaveRoom());
+        this.props.onCallTerminated();
```

Comment:
At this stage, I'd rather keep it simple - especially as I want to quickly uplift this. I'm not entirely happy with the whole flow here - it feels like the feedback view display should be controlled with the `activeRoomStore`, and in the `DesktopRoomConversationView` rather than in the conversation view. However, we got here because we had two call types, and I can't decide at this stage if we should simplify it more or not.